sklearn/metrics/_regression.py

```diff
@@ -117,6 +117,8 @@ def _check_reg_targets(y_true, y_pred, multioutput, dtype="numeric"):
 
     n_outputs = y_true.shape[1]
     allowed_multioutput_str = ("raw_values", "uniform_average", "variance_weighted")
+    # TODO: Remove the first if check when @check_params is implemented for all
+    # functions calling _check_reg_targets.
```

Comment:
You can remove this part. We will find it later when the code coverage will not cover those lines.

Reply:
Sure.